circle.yml

```diff
@@ -1,4 +1,17 @@
 machine:
   node:
     version: 4.3.0
-
+test:
+  override:
+    - ./node_modules/.bin/eslint --quiet:
+        parallel: true
+        files:
+          - client/**/*.js
+          - client/**/*.jsx
+          - server/**/*.js
+          - server/**/*.jsx
+    - NODE_ENV=development make build && NODE_ENV=development ./bin/run-tests:
```

Comment:
That's get us to another unrelated change we discussed today with @nb. We should at some point take advantage of logic build by @mjangda around `config/_shared.json` file and extract config setting that are common for all envs.